File: VB/CustomSeparator/HtmlSeparatorForm.vb
Imports System.Windows.Forms

' ...
Namespace CustomSeparator

    Public Partial Class HtmlSeparatorForm
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Public ReadOnly Property Result As String
            Get
                Return memoEdit1.Text
            End Get
        End Property

        Private Sub memoEdit1_EditValueChanged(ByVal sender As Object, ByVal e As System.EventArgs)
        End Sub
    End Class
End Namespace

File: CS/CustomSeparator/HtmlSeparatorForm.cs
using System.Windows.Forms;
// ...

namespace CustomSeparator {
    public partial class HtmlSeparatorForm : Form {
        public HtmlSeparatorForm() {
            InitializeComponent();
        }

        public string Result { get { return memoEdit1.Text; } }

        private void memoEdit1_EditValueChanged(object sender, System.EventArgs e) {

        }
    }
}
